lib/rbUtil.js

```diff
@@ -173,6 +173,23 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+/**
+ * Copies forwarded headers from restbase to request.
+ * If the same header was already set it takes precedence over
+ * the forwarded header.
+ */
```

Comment:
I proposed yet another option in https://github.com/wikimedia/restbase/pull/272/files#r37483785. Feel free to disregard if this is getting a bit too much ;)